src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -330,4 +335,30 @@ mod test {
             assert_eq!(got, exp);
         }
     }
+
+    #[test]
+    fn test_round() {
+        let tests = vec![
+            (ScalarFuncSig::RoundInt, Datum::I64(1), Datum::I64(1)),
+            (
+                ScalarFuncSig::RoundInt,
+                Datum::I64(i64::MAX),
+                Datum::I64(i64::MAX),
+            ),
+            (
+                ScalarFuncSig::RoundInt,
+                Datum::I64(i64::MIN),
+                Datum::I64(i64::MIN),
+            ),
+        ];
+
+        let mut ctx = EvalContext::new(Arc::new(EvalConfig::new(FLAG_IGNORE_TRUNCATE).unwrap()));
```

Comment:
How about the operator is pushed down without the flag?

Reply:
It seems we needn't consider the flag since the implementation does not relate to it? @hicqu